### PR TITLE
[FIX] Fix usage and address requested changes to rapidtide parser

### PR DESCRIPTION
**Pull request by tsalo**
Closes None.

Changes proposed in this pull request:

- Remove the old CLI help from `usage.rst`, since it conflicted with the auto-generated one.
- Address @bbfrederick's review comments from #46:
    - Remove single-item mutually-exclusive parameter groups from rapidtide parser.
    - Remove incorrect information from "Analysis type" and "Macro" argument group descriptions.
    - Remove values set after parser that are duplicates of values from the parser.
- Remove usage simplifier for rapidtide argparser (this can be reverted if you'd prefer).

### CODE REVIEW

**Comment by tsalo** [comment]
I have a question related to this PR- what are `practicalities.rst` and `theoryofoperation.rst`? I tried to put them in a hidden toctree so that they wouldn't show up on the sidebar and there wouldn't be a "these pages don't show up in any toctree" warning, but for some reason they still showed up. `theoryofoperation` seems to be a copy of `usage.rst`, while `practicalities` seems like a frequently asked questions page.

**Comment by bbfrederick** [comment]
I was looking at those the other day.  Yes, practicalities was the beginning of a FAQ, but I didn't get far with it.  And theoryofoperation.rst is supposed to be a more qualitative view of how the code is designed, and why I made certain choices.  The view from 10000 feet.  It's not supposed to be a word for word copy of usage.rst, which is what it seems to be.  I think maybe I can just put those off to the side and remove them, and when I figure out how I want to write them, I can add them back.

**Comment by tsalo** [comment]
>I think maybe I can just put those off to the side and remove them, and when I figure out how I want to write them, I can add them back.

Okay, that sounds good. I'll remove them in this PR.

> theoryofoperation.rst is supposed to be a more qualitative view of how the code is designed, and why I made certain choices. The view from 10000 feet.

In terms of a 10000 foot view, what do you think of something like [this walkthrough of `tedana`'s workflow](https://tedana.readthedocs.io/en/latest/approach.html)? I was actually thinking that the examples in `usage.rst` would really benefit from figures. Of course, `rapidtide` has enough different tools in it that a single file may sprawl significantly, but perhaps one focusing on the `rapidtide` workflow and one for `happy`, with outputs _using_ the visualization tools (rather than focusing on them), would work well in a walkthrough page.

**Comment by bbfrederick** [comment]
I like the tedana approach - may try something similar.

I think at least for the first go round, if I document rapidtide, happy, and tidepool, that will be the majority of what needs doing.  There are lots of programs in rapidtide, but those three are the ones of widest interest.  Once I get those well documented I can look at the others.

**Comment by bbfrederick** [comment]
Ok, I've started a skeleton of an actual theoryofoperation.rst.  I'll update it over the next few days.

File: rapidtide/workflows/rapidtide_parser.py [hunk 1]
```diff
@@ -46,7 +46,6 @@ def _get_parser():
     parser = argparse.ArgumentParser(
         prog="rapidtide",
         description=("Perform a RIPTiDe time delay analysis on a dataset."),
-        usage="%(prog)s in_file outputname [options]",
```
Comment (tsalo) [inline rapidtide/workflows/rapidtide_parser.py:49]: Is it cool to remove this? I don't know if you want the arguments to show up in the help string, but the patterns in that section are useful for indicating mutually exclusive arguments.

Comment (bbfrederick) [reply rapidtide/workflows/rapidtide_parser.py:49]: Yes, that's fine.  I think that's better (didn't know it would do that!)

Comment (tsalo) [reply rapidtide/workflows/rapidtide_parser.py:49]: Awesome! I think this PR is ready for review then. That was the only change that I wasn't sure you'd want, but please let me know if there's anything in the rest of the PR you want me to change.

Comment (bbfrederick) [reply rapidtide/workflows/rapidtide_parser.py:49]: So I generated a new theoryofoperation.rst after we deleted the one that was there.  This PR won't overwrite that, correct?

Comment (tsalo) [reply rapidtide/workflows/rapidtide_parser.py:49]: I didn't delete `theoryofoperation.rst` in this PR, so there won't be a conflict.

Comment (bbfrederick) [reply rapidtide/workflows/rapidtide_parser.py:49]: 👍

**Pull request merged**